pkg/controller/directpvmigrationprogress/directpvmigrationprogress_controller_suite_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package directpvmigrationprogress
+
+import (
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/konveyor/mig-controller/pkg/apis"
+	"github.com/onsi/gomega"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var cfg *rest.Config
+
+func TestMain(m *testing.M) {
+	//t := &envtest.Environment{
```

Comment:
Remove?

Reply:
that was my local hack to make unit test work, I will uncomment them

Reply:
done